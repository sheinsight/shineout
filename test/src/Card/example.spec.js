import exampleTest from '../../example'

describe('Card[snapshot]', () => {
  exampleTest('Card', {
    ignore: 'example-5-accordion',
  })
})
