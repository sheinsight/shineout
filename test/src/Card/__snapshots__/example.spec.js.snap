// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card[snapshot] Snapshot test: Card[site/pages/components/Card/example-1-base.tsx] 1`] = `
<div>
  <div>
    <div>
      Header
    </div>
    <div>
      Body
    </div>
    <div>
      Footer
    </div>
  </div>
  <div>
    <div>
      Header
    </div>
    <div>
      Body
    </div>
    <div>
      Footer
    </div>
  </div>
</div>
`;

exports[`Card[snapshot] Snapshot test: Card[site/pages/components/Card/example-2-boxshadow.tsx] 1`] = `
<div>
  <div>
    <div>
      Never
    </div>
  </div>
  <div>
    <div>
      Hover
    </div>
  </div>
  <div>
    <div>
      Always
    </div>
  </div>
</div>
`;

exports[`Card[snapshot] Snapshot test: Card[site/pages/components/Card/example-3-form.tsx] 1`] = `
<div>
  <div>
    Header
  </div>
  <div>
    <br />
    <form>
      <div>
        <div>
          User name
        </div>
        <div>
          <label>
            <input />
          </label>
        </div>
      </div>
      <div>
        <div>
          Email
        </div>
        <div>
          <label>
            <input />
          </label>
        </div>
      </div>
    </form>
  </div>
  <div>
    <button>
      <span>
        Submit
      </span>
    </button>
  </div>
</div>
`;

exports[`Card[snapshot] Snapshot test: Card[site/pages/components/Card/example-4-collapse.tsx] 1`] = `
<div>
  <div>
    <span>
      <svg>
        <path />
        <path />
      </svg>
    </span>
    Header
  </div>
  <div>
    <div>
      <div>
        <label>
          <div>
            <div>
              <span>
                <span>
                  Â 
                </span>
              </span>
              <span>
                <svg>
                  <path />
                </svg>
              </span>
            </div>
            <div />
          </div>
        </label>
      </div>
    </div>
  </div>
</div>
`;
